scripts/sysupgrade-tar.sh

```diff
@@ -39,7 +51,7 @@ while [ "$1" ]; do
 done
 
 if [ ! -n "$board" -o ! -r "$kernel" -a  ! -r "$rootfs" -o ! "$outfile" ]; then
-	echo "syntax: $0 [--board boardname] [--kernel kernelimage] [--rootfs rootfs] out"
+	echo 'syntax: $0 [--board boardname] [--alt-boards "alt board list"] [--kernel kernelimage] [--rootfs rootfs] out'
```

Comment:
Using single quotes here will prevent $0 from being evaluated.

Reply:
Right, you are. Thanks